crates/sncast/tests/helpers/fixtures.rs

```diff
@@ -215,15 +224,37 @@ pub async fn invoke_contract(
         calldata,
     };
 
-    let execution = account.execute_v1(vec![call]);
-    let execution = apply_optional(execution, max_fee, ExecutionV1::max_fee);
+    let execution = account.execute_v3(vec![call]);
+    let execution = apply_optional(execution, fee_settings.l1_gas, ExecutionV3::l1_gas);
+    let execution = apply_optional(
+        execution,
+        fee_settings.l1_gas_price,
+        ExecutionV3::l1_gas_price,
+    );
+    let execution = apply_optional(execution, fee_settings.l2_gas, ExecutionV3::l2_gas);
+    let execution = apply_optional(
+        execution,
+        fee_settings.l2_gas_price,
+        ExecutionV3::l2_gas_price,
+    );
+    let execution = apply_optional(
+        execution,
+        fee_settings.l1_data_gas,
+        ExecutionV3::l1_data_gas,
+    );
+    let execution = apply_optional(
+        execution,
+        fee_settings.l1_data_gas_price,
+        ExecutionV3::l1_data_gas_price,
+    );
```

Comment:
If we create a helper for these it can be used here as well

Reply:
Same as above, macro `apply_optional_fields` handles that now.